Mathlib/MeasureTheory/Measure/Typeclasses.lean

```diff
@@ -943,6 +922,35 @@ theorem restrict_toMeasurable_of_sFinite [SFinite μ] (s : Set α) :
       restrict_apply ht, inter_comm t]
 #align measure_theory.measure.restrict_to_measurable_of_sigma_finite MeasureTheory.Measure.restrict_toMeasurable_of_sFinite
 
+theorem iSup_restrict_spanningSets_of_measurableSet [SigmaFinite μ] (hs : MeasurableSet s) :
```

Comment:
Can you add a comment here to say this lemma is superceded by `iSup_restrict_spanningSets`